packages/flutter_tools/bin/xcode_backend.sh

```diff
@@ -214,15 +220,7 @@ EmbedFlutterFrameworks() {
 
   # Embed the actual Flutter.framework that the Flutter app expects to run against,
   # which could be a local build or an arch/type specific build.
-
-  # Copy Xcode behavior and don't copy over headers or modules.
-  RunCommand rsync -av --delete --filter "- .DS_Store" --filter "- Headers" --filter "- Modules" "${BUILT_PRODUCTS_DIR}/Flutter.framework" "${xcode_frameworks_dir}/"
-
-  # Sign the binaries we moved.
-  if [[ -n "${EXPANDED_CODE_SIGN_IDENTITY:-}" ]]; then
-    RunCommand codesign --force --verbose --sign "${EXPANDED_CODE_SIGN_IDENTITY}" -- "${xcode_frameworks_dir}/App.framework/App"
-    RunCommand codesign --force --verbose --sign "${EXPANDED_CODE_SIGN_IDENTITY}" -- "${xcode_frameworks_dir}/Flutter.framework/Flutter"
-  fi
+  RunCommand rsync -av --delete --filter "- .DS_Store" "${BUILT_PRODUCTS_DIR}/Flutter.framework" "${xcode_frameworks_dir}/"
```

Comment:
Had to stop filtering out `Headers` and `Modules` directories or the codesigning done before the copy became invalid.